common-rendering/src/editorialPalette/background.ts

```diff
@@ -596,6 +596,53 @@ const articleContent = (format: ArticleFormat): string => {
 	}
 };
 
+
+
+const byline = (format: ArticleFormat): Colour => {
+	if (format.design === ArticleDesign.Analysis) {
+		switch (format.theme) {
+			case ArticlePillar.Sport:
+				return sport[300];
+			case ArticlePillar.Culture:
+				return culture[300];
+			case ArticlePillar.Opinion:
+				return opinion[300];
+			case ArticlePillar.Lifestyle:
+				return lifestyle[300];
+			case ArticleSpecial.Labs:
+				return labs[300];
+			case ArticleSpecial.SpecialReport:
+				return brandAlt[300];
+			case ArticlePillar.News:
+			default:
+				return news[300];
+		}
+	}
+
+	return neutral[100];
+};
+
+const bylineDark = (format: ArticleFormat): Colour => {
+	if (format.design === ArticleDesign.Analysis) {
+		switch (format.theme) {
+			case ArticlePillar.Sport:
+				return sport[500];
+			case ArticlePillar.Culture:
+				return culture[500];
+			case ArticlePillar.Opinion:
+				return opinion[500];
+			case ArticlePillar.Lifestyle:
+				return lifestyle[500];
+			case ArticlePillar.News:
+			default:
+				return news[500];
+		}
+	}
+
+	return neutral[0];
+};
```

Comment:
I think you may instead want to add these to `text.bylineAnchor` and `text.bylineAnchorDark`?

Reply:
Good spot, thanks! I'll amend this